tailwind.config.js

```diff
@@ -71,7 +67,7 @@ module.exports = {
         se: '23.5rem',
         tablet: '31.25rem',
         md: '37.5rem',
-        sm: '40rem',
+        sm: '42.1875rem',
```

Comment:
Should we round to two digits after a point? I think, we should

Reply:
No, I need exactly `675px` which is `'42.1875rem'`